integrations/third_party/sample_integration/actions/enrich_entity_action_example.py

```diff
@@ -0,0 +1,85 @@
+from TIPCommon.base.action import EntityTypesEnum
+from TIPCommon.base.action.data_models import DataTable, Entity
+from TIPCommon.extraction import extract_action_param
+from TIPCommon.transformation import add_prefix_to_dict, construct_csv
+from TIPCommon.validation import ParameterValidator
+from soar_sdk.SiemplifyUtils import unix_now
+
+from ..core.base_action import BaseAction
+from ..core.constants import (
+    ENRICH_ENTITY_ACTION_EXAMPLE_SCRIPT_NAME,
+    SupportedEntitiesEnum,
+)
+
+DEFAULT_ENTITY_TYPE = SupportedEntitiesEnum.ALL.value
+
+SUCCESS_MESSAGE = "Successfully enriched the following entities: {}"
+NO_ENTITIES_MESSAGE = "No eligible entities were found in the scope of the Alert."
+
+
+class EnrichEntityActionExample(BaseAction):
+    def __init__(self):
+        super().__init__(ENRICH_ENTITY_ACTION_EXAMPLE_SCRIPT_NAME)
+        self.enriched_entities: list[str] = []
+        self.error_output_message: str = (
+            f'Error executing action "{ENRICH_ENTITY_ACTION_EXAMPLE_SCRIPT_NAME}".'
+        )
+
+    def _extract_action_parameters(self):
+        self.params.entity_type = extract_action_param(
+            self.soar_action,
+            param_name="Entity Type",
+            default_value=DEFAULT_ENTITY_TYPE,
+            print_value=True,
+        )
+
+    def _validate_params(self):
```

Comment:
```suggestion
    def _validate_params(self) -> None:
```

type checkers like mypy consider this as not annotated and skip it. Please add explicit return types even for None values. It can also helps  with return type issues when coding if something is returned but it doesn't expect to